winforms/test_probes/widgets/probe_slider.py

```diff
@@ -0,0 +1,20 @@
+import System.Windows.Forms
+
+from .probe_base import SimpleProbe
+
+
+class SliderProbe(SimpleProbe):
+    native_class = System.Windows.Forms.TrackBar
+
+    @property
+    def position(self):
+        min, max = self._range
+        return (self.native.Value - min) / (max - min)
+
+    def change(self, position):
+        min, max = self._range
+        self.native.Value = min + round(position * (max - min))
```

Comment:
Methods to trigger events are named the same as the corresponding event handler.

Reply:
Is that purely a convention, or is there a functional consequence?

Reply:
Just a convention.